.github/styles/vocab.txt

```diff
@@ -276,6 +278,7 @@ validators
 varchar
 winston
 www
+xlarge
```

Comment:
Would say do the same with this btw, `BoxUsage:m3.xlarge(Hrs)` instead of adding to vocab